shared/actions/settings.js

```diff
@@ -66,6 +93,135 @@ function * saveNotificationsSaga (): SagaGenerator<any, any> {
   }
 }
 
+function * reclaimInviteSaga (invitesReclaimAction: InvitesReclaim): SagaGenerator<any, any> {
+  const {inviteId} = invitesReclaimAction.payload
+  try {
+    yield call(apiserverPostRpcPromise, {
+      param: {
+        endpoint: 'cancel_invitation',
+        args: [{key: 'invitation_id', value: inviteId}],
+      },
+    })
+    yield put(({
+      type: Constants.invitesReclaimed,
+      payload: undefined,
+      error: false,
```

Comment:
in my mind its all success case unless it has error=true. I'm fine with making payload optional (we follow FSA and it says it can be optional)